gocourse05/camera/infrared.go

```diff
@@ -0,0 +1,60 @@
+package camera
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+func NewInfraredCameraData(id int, animal, movement string) Data {
+	return Data{
+		ID:       id,
+		Animal:   animal + "_" + uuid.NewString(),
+		Movement: movement,
+	}
+}
+
+type InfraredCamera struct {
+	ID        int
+	Name      string
+	Data      []Data
```

Comment:
Пропоную зробити поля неекспортованими для інкапсуляції:
```suggestion
	id        int
	name      string
	data      []Data
```

Для доступу до `id` додати метод `InfraredCamera.ID() int`